src/nft/hooks/useSellAsset.ts

```diff
@@ -0,0 +1,148 @@
+/* tslint:disable */
```

Comment:
You shouldn't be disabling tslint and eslint for an entire file - it should only be targeted, with a comment explicitly stating why it is disabled.